specification/vmware/resource-manager/Microsoft.AVS/stable/2023-09-01/vmware.json

```diff
@@ -8539,7 +8539,8 @@
             "description": "is updating"
           }
         ]
-      }
+      },
+      "readOnly": true
```

Comment:
Those changes are a bug fix where we didn't detect those unions as provisioning state unions and you had `use-readonly-provisioning-state` option